CONTRIBUTING.md

```diff
@@ -24,6 +24,29 @@ pre-installed dependencies, running web server and rollup watching for changes.
 
 [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/from-referrer/)
 
+### Making a PR through the online-setup
+
+- Fork the repo by pressing `F1` and then type in fork and then hit enter.
```

Comment:
Is there not a menu item here that F1 is a shortcut for?

Reply:
Thanks! There is a menu item I will shortly update the CONTRIBUTING.md:
![image](https://user-images.githubusercontent.com/46004116/76392493-75727880-6393-11ea-8939-41e6c36477de.png)